grakn-test/src/test/java/ai/grakn/generator/ConceptIds.java

```diff
@@ -22,7 +22,7 @@
 import ai.grakn.concept.ConceptId;
 
 /**
- * Generator that generates totally concept IDs
+ * Generator that generates totally random concept IDs
  */
 public class ConceptIds extends AbstractGenerator<ConceptId> {
```

Comment:
Can we not move all these generator classes to `grakn-test-tools` instead? By moving them to `grakn-test` it means that they can only be used in that module (very bad in my opinion), but by moving them to `grakn-test-tools` they can be used in any module (except `grakn-graph`).

I just feel that moving the generators to `grakn-test` is a step towards making that a monolithic test module again.